examples/opengl_underlay/main.cpp

```diff
@@ -169,8 +169,8 @@ int main()
     if (auto error = app->window().set_rendering_notifier(OpenGLUnderlay(app))) {
         if (*error == slint::SetRenderingNotifierError::Unsupported) {
             fprintf(stderr,
-                    "This example requires the use of the GL backend. Please run with the "
-                    "environment variable SIXTYFPS_BACKEND=GL set.\n");
+                    "This example requires the use of a GL renderer. Please run with the "
+                    "environment variable SLINT_BACKEND=winit set.\n");
```

Comment:
please double check

Reply:
I think in practice this is kind of ok. What we should have is sort this out at compile-time in this example, via CMake features set here and C++ API to explicitly select the renderer (like Qt does). But that's for another day :)